lib/components/dropdown-item-button.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+    <button class="dropdown-item"
+            type="button"
+            role="menuitem"
+            :disabled="disabled"
+            @click="onClick">
+        <slot></slot>
+    </button>
+</template>
+
+<script>
+export default {
+    props: {
+        disabled: {
+            type: Boolean,
+            default: false
+        }
+    },
+    methods: {
+        onClick(e) {
+            this.$parent.$el.click(e);
```

Comment:
Alright, I think I might have it figured out—`this.$root.$on('clicked::link', listener);`